dotcom-rendering/src/amp/components/Ad.tsx

```diff
@@ -46,17 +46,67 @@ const mapAdTargeting = (adTargeting: AdTargeting): AdTargetParam[] => {
 	return adTargetingMapped;
 };
 
-const realTimeConfig = (
+// Variants for the Prebid server test
+// Assign each variant 4 groups e.g. 33.3% of content types each
+const variants = {
+	control: new Set([0, 1, 2, 3]),
+	'relevant-yield': new Set([4, 5, 6, 7]),
+	pubmatic: new Set([8, 9, 10, 11]),
+};
+
+// Determine participation in a variant from group
+const isInVariant = (
+	variantName: 'control' | 'relevant-yield' | 'pubmatic',
+	group: number | undefined,
+) => group !== undefined && variants[variantName].has(group);
+
+const useRealTimeConfig = (
 	usePrebid: boolean,
 	usePermutive: boolean,
 	useAmazon: boolean,
-	placementID: number,
+	{ placementId, tagId, profileId, pubId }: RTCParameters,
 ): string => {
+	const { group } = useContentABTestGroup();
```

Comment:
This is where we determine group participation (via the `useContentABTestGroup` hook) and based upon which group the content is in we build a different RTC to place inside the AMP element.